### ISSUE TITLE
Clean up documentation

### ISSUE THREAD

**Issue by cvvergara** [pgRouting/pgrouting#748]
The documentation has this section

```
Tutorial

        Getting started
        Routing Topology for an overview of a topology for routing algorithms.
        Graph Analytics for an overview of the analysis of a graph.
        Dictionary of columns & Custom Query that is used in the routing algorithms.  
        Performance Tips to improve your performance.
        User’s Recipes List
        Developer’s Guide
```

* Dictionary of columns & Custom Query that is used in the routing algorithms
  * its an attempt to solve https://github.com/pgRouting/pgrouting/issues/400
  * So put it somewhere else maybe with a different name.

* User’s Recipes List
  * Can be moved to the wiki

* Developer’s Guide
  * Content can be moved to [doxygen](http://docs.pgrouting.org/doxy/2.4/index.html) documentation which is a more complete developer's documentation

* As for this ones:
        Getting started
        Routing Topology for an overview of a topology for routing algorithms.
        Graph Analytics for an overview of the analysis of a graph.

Its mostly about "checking" the topology, here there is no data, examples are based on unavailable data. (the examples on the documentation of the topology functions look more complete)

basically what I am saying is:
delete the so called "tutorial" from the documentation.

**Comment by Neel-Shah-29** [comment]
Hello is there anyone working on this issue ? If not than i am interested in working on this issue kindly assigned it to me and guide me once if possible
